### DISCUSSION TITLE
Support fallback to source locale

### DISCUSSION THREAD

**Discussion by yunsii** [lingui/js-lingui#1791] [General]
**Is your feature request related to a problem? Please describe.**

In a big project, if someone do not extract and translate messages, it will be render as message id directly in production env. It is unreasonable I think.

**Describe proposed solution**

Is it possible to use source locale message to render after build? Or report un-translated messages at some time.

**Comment by timofei-iatsenko** [comment]
Add extracting to your build step before building app, something like `build: "lingui extract && next build"` this way catalogs would always be fulfilled with original messages.

**Reply by yunsii** [reply]
It is only works for source locale, fulfilled with original messages. But not work for others locales.
